docs/rest-apis/gateway/schemas.md

```
@@ -3762,6 +3858,127 @@ and
+<a id="schema_WebhookSecretCreationRequest"></a>
+<a id="tocSwebhooksecretcreationrequest"></a>
+<a id="tocswebhooksecretcreationrequest"></a>
+
+```json
+{
+  "displayName": "GitHub Webhook"
+}
+
+```
+
+### Properties
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)